openjdk/build.gradle

```diff
@@ -153,19 +156,17 @@ model {
 
     tasks {
         $.binaries.withType(SharedLibraryBinarySpec).each {
-            def archName = it.targetPlatform.architecture.name.replaceAll('-', '_')
             def source = it.sharedLibraryFile
 
             // Copies the native library to a resource location that will be included in the jar.
-            task "copyNativeLib${archName}"(type: Copy) {
+            task "copySharedNativeLib${osdetector.arch}"(type: Copy) {
                 from source
-                rename '(.+)(\\.[^\\.]+)', "\$1-$archName\$2"
```

Comment:
The output format doesn't add the build target's platform type as our JNI loading class expects, so removing this line won't work.